deploy/manifests/dev/us-east-2/tenant/storetheindex/indexstar/deployment.yaml

```diff
@@ -16,15 +16,13 @@ spec:
             - '--translateNonStreaming'
             # Use service names local to the namespace over HTTP to avoid
             # TLS handshake overhead.
-            - '--backends=http://ber-indexer:3000/'
-            - '--backends=http://cali-indexer:3000/'
```

Comment:
I think these changes would break the web front end in dev.

Reply:
I changed this to ago, even though ago cannot handle encrypted or non-encrypted queries.